azure_monitor/tests/metrics/test_metrics.py

```diff
@@ -74,7 +88,9 @@ def test_export(self,):
         record = MetricRecord(
             CounterAggregator(), self._test_label_set, self._test_metric
         )
-        exporter = AzureMonitorMetricsExporter()
+        exporter = AzureMonitorMetricsExporter(
+            storage_path=os.path.join(TEST_FOLDER, self.id())
```

Comment:
Is it possible to clean default path instead of sending custom one for all instances?

Reply:
I agree this is better. I'll make this change.